app/build.gradle

```diff
@@ -348,13 +326,38 @@ android {
         webkit {
             dimension "backend"
         }
+
+        // Stores flavor
+        generic {
+            dimension "store"
+        }
+
+        appLab {
+            dimension "store"
+            externalNativeBuild {
+                cmake {
+                    cppFlags "-DSTORE_BUILD", "-DMETA_APP_ID=4812663595466206"
```

Comment:
In the App Lab build we also need the app ID of  the version of Wolvic that is in the Store.

We use that other ID to open the correct Store page when the user wants to install the new release.

Maybe we could add that second value in a `buildConfigField`?

Reply:
It should be fine to hardcode it in the Java code.